nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server.hpp

```diff
@@ -133,6 +150,24 @@ class BtActionServer
     return action_server_->get_current_goal();
   }
 
+  /**
+   * @brief Wrapper function to get pending goal
+   * @return Shared pointer to pending action goal
+   */
+  const std::shared_ptr<const typename ActionT::Goal> getPendingGoal() const
+  {
+    return action_server_->get_pending_goal();
+  }
+
+  /**
+   * @brief Wrapper function to check if preempt is requested
+   * @return bool true if preempt is requested, false otherwise
+   */
+  bool isPreemptRequested()
```

Comment:
This function isn't used anymore. I totally agree its valuable, but dead code is exactly why costmap_2d is such a mess :laughing: We can readd it if its needed at any time